state/candidates/index.ts

```diff
@@ -11,6 +11,7 @@ const candidateApi = {
 
     checkResponse(response);
 
+    // eslint-disable-next-line total-functions/no-unsafe-type-assertion
```

Comment:
But the cost of fixing that false positive is that all your unsafe type assertions get picked up now :P